<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Library Management System</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="Name">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Name:</string>
    </property>
   </widget>
   <widget class="QLabel" name="ISBN">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>ISBN:</string>
    </property>
   </widget>
   <widget class="QLabel" name="PuchaseDate">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Purchased On:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Category">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Category:</string>
    </property>
   </widget>
   <widget class="QLabel" name="SubCategory">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>SubCategory:</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="AuthorsGroup">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>461</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string>Authors</string>
    </property>
    <widget class="QLabel" name="AuthorName">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Author Name</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="AuthorNameEdit">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>251</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="AddAuthor">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="AuthorNames">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>50</y>
       <width>251</width>
       <height>151</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="TypeGroup">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>450</y>
      <width>191</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>Type</string>
    </property>
    <widget class="QRadioButton" name="RBbutton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>111</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Reference Book</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="TBbutton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>101</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Text Book</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="Jbutton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>101</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Journal</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="IssuanceGroup">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>450</y>
      <width>261</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>Issuance</string>
    </property>
    <widget class="QCheckBox" name="issuedCheck">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Issued</string>
     </property>
    </widget>
    <widget class="QLabel" name="IssuedBy">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Issued By:</string>
     </property>
    </widget>
    <widget class="QLabel" name="IssuedOn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Issued On:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="issuedEdit">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>70</y>
       <width>181</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QDateEdit" name="IssueDate">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>110</y>
       <width>181</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="OkayButton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>620</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Okay</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CloseButton">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>620</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="NameLine">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>50</y>
      <width>351</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="ISBNLine">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>80</y>
      <width>351</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QDateEdit" name="purchaseDateEdit">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>110</y>
      <width>351</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="categories">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>140</y>
      <width>351</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Database Systems</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>OOP</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Artificial Intelligence</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="subcategories">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>170</y>
      <width>351</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>ERD</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>SQL</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>OLAP</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Data Mining</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>C++</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Java</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Machine Learning</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Robotics</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Computer Vision</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
